packages/platforms/delta/x86-64/x86-64-delta-ag8032/onlp/builds/src/module/src/thermali.c

```diff
@@ -203,44 +210,46 @@ onlp_thermali_info_get(onlp_oid_t id, onlp_thermal_info_t* info)
 	// Get value
 	error = 0;
 	if (info->caps & ONLP_THERMAL_CAPS_GET_TEMPERATURE) {
-		if (plat_os_file_read_int(&value, thermal->temp_get_path, NULL) < 0)
+		if (plat_os_file_read_int(&value, thermal->temp_get_path, NULL) < 0) {
+			AIM_LOG_ERROR("plat_os_file_read_int failed for ONLP_THERMAL_CAPS_GET_TEMPERATURE path=%s", thermal->temp_get_path);
```

Comment:
there is no need the AIM_LOG_ERROR, otherwise it could output the errror log when the psu mode was not plugged in, and unplugged status of the psu is the normal status.